deploy/overlays/dev-template/config.yaml

```diff
@@ -4,7 +4,12 @@ kind: JBSConfig
 metadata:
   name: jvm-build-config
 spec:
-  owner: QUAY_USERNAME
+  # TODO: Should this be qualified by 'registry' ? Same for ci-template?
+  registry:
+    owner: QUAY_USERNAME
```

Comment:
Given scoping changed to explicitly state registry in https://github.com/redhat-appstudio/jvm-build-service/blob/main/pkg/apis/jvmbuildservice/v1alpha1/jbsconfig_types.go#L43 I think this needs an update?